bundles/org.openhab.binding.qbus/README.md

```diff
@@ -0,0 +1,110 @@
+# Qbus Binding
+
+![Qbus Logo](doc/Logo.JPG)
+
+This binding for Qbus communicates for all controllers of the Qbus home automation system.
+
+More information about Qbus can be found here:
+[Qbus](https://qbus.be)
+
+This is the link to the [Qbus forum](https://qbusforum.be). This forum is mainly in dutch and you can find a lot of information about the pre testings af this binding and offers a way to communicate with other users.
+
+We also host a site which contains a [manual](https://manualoh.schockaert.tk/) where you can find lot's of information.
```

Comment:
Can you mention that this is in dutch?

Is this related to openHAB or to Qbus in general?